packages/dataviews/src/dataviews-layouts/grid/preview-size-picker.tsx

```diff
@@ -34,10 +38,6 @@ export default function PreviewSizePicker() {
 	const context = useContext( DataViewsContext );
 	const view = context.view as ViewGrid;
 
-	if ( context.containerWidth < 588 ) {
-		return null;
```

Comment:
I was curious about this removal, and then realised that of course now we can see that there's multiple options available at smaller container sizes 👍 

<img width="397" height="604" alt="image" src="https://github.com/user-attachments/assets/873b3760-9d44-4a31-99c6-104a8b072c9b" />